sql/ddl/alter/add-column.md

```diff
@@ -48,23 +48,23 @@ aspects:
 
 The `ALTER TABLE` command lets you modify table constraints and column definitions after you've defined your table.
 
-If you'd like to add a new column to a certain table, we can use the `ALTER TABLE` statement:
+For example, to add a new column to a certain table, you can use:
 
 ```sql
 ALTER TABLE table_name
 ADD column_name datatype;
 ```
 
-You must always specify the column name and its data type, but there are some other optional arguments you can use. For example, you can add a `CHECK` constraint, a `DEFAULT` value, a `NOT NULL` clause, or even set the column as a `PRIMARY KEY` (that is, if there isn't a `PRIMARY KEY` already defined).
+You must always specify the column name and its data type, with some other option arguments being available.You can add a `CHECK` constraint, a `DEFAULT` value, a `NOT NULL` clause, or even set the column as a `PRIMARY KEY` (that is, if there isn't a `PRIMARY KEY` already defined).
```

Comment:
"option" should still say "optional". Personally, I'd also leave the "For example" part, the sentence flow feels better that way

Maybe say:

```suggestion
You must always specify the column name and its data type, with some other option arguments being available. For example, you can add a `CHECK` constraint, a `DEFAULT` value, a `NOT NULL` clause, or even set the column as a `PRIMARY KEY` (that is, if there isn't a `PRIMARY KEY` already defined).
```